lti_consumer/lti_consumer.py

```diff
@@ -709,6 +677,18 @@ def prefixed_custom_parameters(self):
                     param_name = 'custom_' + param_name
 
                 custom_parameters[six.text_type(param_name)] = six.text_type(param_value)
+
+        custom_parameters[six.text_type('custom_component_display_name')] = six.text_type(self.display_name)
```

Comment:
Do we need to be using six in this repo still?  My understanding was that it was for ensuring python 2-3 compatibility.  With the `lti_embed` syntax change above, we're no longer compatible with python 2 anyway, right?

Reply:
@davidjoy Happy to remove `six` if we are comfortable with the repo not being python 2 compatible anymore. In this particular case though, if people are only using the `LtiConsumerXBlock`, everything they have access to from that class is still in fact python 2 compatible. I'm not sure if we are comfortable with partial compatibility or not though.

I feel like the options we likely have available are:
- Remove `six`, bump the major version, and not longer support Python 2 compatibility
- Modify the `lti_embed` syntax to be Python 2 compatible
- Be comfortable with partial compatibility (and possibly make this a minor version bump instead of a patch version bump)

Please advise to your desired course of action. I think it probably makes sense for @giovannicimolin to weigh in on this as well

Reply:
I think it makes sense to break compatibility and make a major release, here is why:
-This block comes pre-installed with the platform, and it's unclear if anyone upgrades it to a newer version. 
-This version will come with Koa, and I doubt Python 2 will be supported for that.
- Python 2 itself isn't supported anymore.

Reply:
Sounds good, I will bump the major version. `six` is used quite extensively in this repo though, so I'm inclined not to clutter this PR up with the removal of `six`. I will remove it from any code that was touched in this PR though.

Reply:
👍 On this.  Agree on the major version bump, especially after conferring with others on the platform team here.  Agree that removing six can happen separately.